src/lib/analytics/index.js

```diff
@@ -2,6 +2,13 @@
  * Internal dependencies
  */
 import AnalyticsClientStub from './clients/stub';
+import { version } from '../../../package.json';
+
+const client_info = {
+	node_version: process.version,
+	os: process.platform,
+	cli_version: version,
```

Comment:
We can just grab these values from the `env` module: https://github.com/Automattic/vip/blob/747b6a49ee1b864093a3df7ea5a11afc13f57124/src/lib/env.js

For `os` especially, it would be nice to include the version as well.

/cc @simonwheatley for verification on prop names and the data we're passing through.

Reply:
> For `os` especially, it would be nice to include the version as well.

+1

@elazzabi Can you give an example data dump of the values for the props being added, please?

Reply:
according to tracks, no information is being passed in the `props`! i double checked in local, and indeed, nothing is passed. something is wrong.

this PR will add the following to every request: 
```
{ cli_version: '1.2.0-0',
  os_name: 'darwin',
  os_version: '17.7.0',
  node_version: 'v8.11.2' }
```

need to dig deeper to see why other things aren't being passed, like user id...

Reply:
Sorry for the delay in responding, that additional data looks good to me 👍

Reply:
good to go @mjangda ?